src/components/HomeExperience.module.css

```
@@ -0,0 +1,220 @@
+.site {
+  --ink: #102d2c;
+  --deep: #0a292b;
+  --paper: #efe9d7;
+  --sky: #b8e0df;
+  --coral: #ee3c78;
+  --rust: #b84031;
+  --acid: #d7e34d;
+  background: var(--paper);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)